### PR TITLE
mfw forget to initialize a list

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl:
fix: conveyor belts were definitely not broken, nothing to see here gamers
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Goofball be like

Boop beep bop beep boop bop beep boop ppprrrrrrrrrrrrrrttttttt prrt eet prtt eeeeeeeeeeeeeee prrrrrrrrrrrrrrrrtttt grrrr wee ooo wee ooo wee ffffffffffffffffffffffffffffff fffffffffffffffff

**Comment by qwertyquerty** [comment]
Add CL for downstreams

**Comment by Iamgoofball** [comment]
done

**Comment by ike709** [comment]
smh it doesn't even pass checks

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 
 /obj/machinery/conveyor/auto/Initialize(mapload, newdir)
 	. = ..()
+	affected = list()
```
Comment (ike709) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
	affecting = list()
```

**Pull request merged**